string name
---
bool success
string message
poi_state_server/POIInfo info

